### PR TITLE
Update Figma file urls

### PR DESCRIPTION
**Pull request by tuentisre**
New version of icons exported from [Mistica Icons](https://www.figma.com/design/JXy7Y07eb0Axg0ThVRWKju/Default-Icons-Set?node-id=0-71&t=1nybaZ1fzHqRq80C-1), [O2](https://www.figma.com/file/wHTqJ7KDhGKrNSNpmMb9nW/?node-id=611%3A3298), [O2-new](https://www.figma.com/design/CjvgrHEIycSQ6exznxnFXT/O2?m=auto&node-id=4-0&t=0HBFuR3VADUhh5Zr-1), [Blau](https://www.figma.com/design/czemeClWRGBI8oF7caNa5m/Blau?node-id=0-1&node-type=canvas&t=CM2TIEBRzYVgRX5A-0) & [Vivo](https://www.figma.com/design/EApRpjaTyUOwW5VQU2ZqgP/Vivo?node-id=4-0&node-type=canvas&t=pnjxXOX7B7hd3BmJ-0) 
 - [ ] [Review readme import-figma-icons](https://github.com/Telefonica/mistica-icons/tree/import-figma-icons)

### CODE REVIEW

File: icons/o2-new/light/thumb-down-light.svg
Comment (andredelramo-hm) [inline icons/o2-new/light/thumb-down-light.svg:1]: new icon

File: icons/o2-new/regular/thumb-down-regular.svg
Comment (andredelramo-hm) [inline icons/o2-new/regular/thumb-down-regular.svg:1]: new icon

File: icons/o2-new/filled/thumb-down-filled.svg
Comment (andredelramo-hm) [inline icons/o2-new/filled/thumb-down-filled.svg:1]: new icon

**Review by andredelramo-hm — CHANGES_REQUESTED** [review]

File: icons/o2-new/filled/thumb-down-filled.svg
Comment (yceballost) [reply icons/o2-new/filled/thumb-down-filled.svg:1]: expected

File: icons/o2-new/regular/thumb-down-regular.svg
Comment (yceballost) [reply icons/o2-new/regular/thumb-down-regular.svg:1]: expected

File: icons/o2-new/light/thumb-down-light.svg
Comment (yceballost) [reply icons/o2-new/light/thumb-down-light.svg:1]: expected

**Review by andredelramo-hm — APPROVED** [review]

**Pull request merged**